src/content/en/updates/2017/09/sensors-for-the-web.md

```diff
@@ -1,6 +1,6 @@
 project_path: /web/_project.yaml
 book_path: /web/updates/_book.yaml
-description: Generic Sensor API is available for Origin Trials in Chrome 62.
+description: Generic Sensor API is available for Origin Trials in Chrome 63.
 
 {# wf_updated_on: 2017-09-22 #}
```

Comment:
Please update the `{# wf_updated_up #}` tag

Reply:
done